src/MPAGSCipher/RunCaesarCipher.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+#include <string>
+#include <vector>
+#include <cmath>
+
+#include "RunCaesarCipher.hpp"
+
+std::string runCaesarCipher(const std::string& inputText, const std::size_t key, const bool encrypt, const bool decrypt){
+
+    /*
+        encrypts or decrypts an input string using a Caesar Cipher
+
+        const std::string& inputText: input string which will be encrypted or decrypted
+        const std::size_t key: the key to the Caesar Cipher
+        const bool encrypt: encyrption condition
+        const bool decrypt: decryption condition
+
+        return: cipherOut -- string which has been encrypted or decrypted
+    */
+
+    //create the alphabet container and output string
+    const std::vector<char> alphabet = {'A','B','C','D','E','F','G','H','I',
+                                        'J','K','L','M','N','O','R','Q','R',
+                                        'S','T','U','V','W','X','Y','Z'};
+    std::string cipherOut{""};
+    //initilaised some variables for use later
+    char newChar{'a'};
+    int indexpos{0};
+    int remainder{0};
+
+
+    //loop over each character in the text
+    for(std::size_t i{0}; i < inputText.length();i++){
+
+        for(std::size_t j{0}; j < alphabet.size();j++){ //checks each value in the input string against the alphabet container 
+            if (inputText[i]==alphabet[j]){ //checks for match between character and alphabet letter
+                indexpos = j;   //if match then assign the vector index to indexpos variable
+                continue;   //exit loop as we don't need to do look anymore
```

Comment:
I thikn you mean `break` here - `continue` will just skip to the next iteration, not exit the loop.